string name
uint8 id
geometry_msgs/Pose head_pose